test/xpu_api/random/statistics_tests/bernoulli_distribution_test.pass.cpp

```diff
@@ -50,7 +50,7 @@ test(sycl::queue& queue, double p, int nsamples)
 {
 
     // memory allocation
-    bool samples[nsamples];
+    bool* samples = new bool[nsamples];
```

Comment:
maybe std::vector is better here?

Reply:
Unfortunately, no. We can not get pointer to data using `vector<bool>`. If we speak about `vector<int>` it needs a data conversion from `sycl::vec<bool>` to `sycl::vec<int>` that complicates the code of tests